packages/utilities/gel-foundations/src/scripts/arabic.js

```diff
@@ -0,0 +1,88 @@
+const arabicTypography = {
```

Comment:
I'd suggest the filename is enough to denote the service name, and that this should be pure config (i.e. `arabic.js` -> `arabic.json`, make it a pure JSON file)

Reply:
@ChrisBAshton Could we agree on this with the rest of the team, please? Because we defined them initially as JSON files but replaced then to JS after [this](https://github.com/bbc/psammead/pull/353#pullrequestreview-215505449) discussion

Reply:
Ha, didn't know that! Apologies for the mixed messages. I'm happy leaving as-is and assume others have looked into the implications of it more than me.